tests/tests.rs

```diff
@@ -2156,18 +2156,33 @@ fn test_casr_cluster_d() {
 
     assert!(output.status.success());
 
-    let dirvec = match fs::read_dir(&paths[1]) {
+    let mut dirvec = match fs::read_dir(&paths[1]) {
         Ok(vec) => vec,
         Err(why) => {
             panic!("{:?}", why.kind());
         }
     };
 
+    let casrep = dirvec.next().unwrap().unwrap().path();
```

Comment:
I think, comment is needed why you delete one report.

Reply:
done